src/translations/data.json

```diff
@@ -1871,7 +1871,7 @@
     "sk": "1.jed.",
     "sl": "1.ed.",
     "hr": "1.jed.",
-    "sr": "1.јед",
+    "sr": "1. л. јед",
```

Comment:
Jest li to prinicipialno? Sde imajemo uzku (ne široku) kolonu, ne jest li razumlivo 1.jed. kako 1 lico, jednina? Ili može togda napisali 1l.jed. ?

Reply:
ok. Pozdrav Danko. možno izkydnuti. togda možno ostaviti 1. jed., 2. jed itd.

Reply:
Ty možeš izpraviti sam v ramah svojej větky (branch): https://github.com/Bardzobardzowredna/interslavic/blob/patch-1/src/translations/data.json